README.md

```diff
@@ -26,7 +26,12 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+ def offer_rose(person)
+   puts ' Would you take this rose and help out an old beggar, #{person}?'
+   end
+end
+
+def offer_rose(young_prince)
```

Comment:
The argument passed in should be a string (e.g., `"young_prince"`). As written, you are passing in an `young_prince` variable that has not been defined.